keras/backend/tensorflow_backend.py

```diff
@@ -332,14 +636,55 @@ def random_normal_variable(shape, mean, scale, dtype=_FLOATX,
 
 
 def count_params(x):
-    '''Returns the number of scalars in a tensor.
+    '''Returns the number of scalars in a Keras variable.
+
+    # Arguments
+        x: Keras variable.
+
+    # Returns
+        Integer, the number of scalars in `x`.
+
+    # Example
+    ```python
+        >>> var = K.zeros((2,3))
+        >>> K.count_params(var)
+        6
+        >>> K.eval(var)
+        array([[ 0.,  0.,  0.],
+               [ 0.,  0.,  0.]], dtype=float32)
+    ```
     '''
     shape = x.get_shape()
     return np.prod([shape[i]._value for i in range(len(shape))])
 
 
 def cast(x, dtype):
-    '''Casts a tensor to a different dtype.
+    '''Casts a Keras variable to a different dtype.
```

Comment:
This is primarily meant for tensor casting, not variable casting. Albeit you could cast a variable (thus returning a tensor).